omod/src/main/webapp/pages/login.gsp

```diff
@@ -41,6 +37,15 @@ ${ ui.includeFragment("referenceapplication", "infoAndErrorMessages") }
             updateSelectedOption();
         });
         
+        jQuery('#login-form').submit(function(e) {
+        	var sessionLocationVal = jQuery('#sessionLocationInput').val();
+        	
+        	if (!sessionLocationVal) {
```

Comment:
@ivange94 did you intentionally leave out some of the tests that the line 27 above does?